client/src/nonblocking/tpu_client.rs

```diff
@@ -2,7 +2,6 @@ pub use solana_tpu_client::nonblocking::tpu_client::{LeaderTpuService, TpuSender
 use {
```

Comment:
Can we actually remove the duplicated client/src/tpu_client.rs and client/src/nonblocking/tpu_client.rs here and just re-export from tpu-client, now that the generics are removed?
Or does that not work because of the change in ctor type signature?